firmware/ec/src/devices/eth_sw.c

```diff
@@ -98,6 +99,7 @@ uint16_t get_interrupt_status(uint8_t port)
 /*****************************************************************************
  * Internal IRQ handler - reads in triggered interrupts and dispatches CBs
  *****************************************************************************/
+#if 0
```

Comment:
Why was this commented out?

Reply:
The function is not used, therefore we get a warning about an unused function.  It seems like it could be useful at some future time, which is why I #if 0'd it.

Reply:
The function is not used, therefore we get a warning about an unused function. It seems like it could be useful at some future time, which is why I #if 0'd it.

Reply:
Added comment to clarify intention